contracts/State.sol

```diff
@@ -92,12 +86,12 @@ contract State {
         uint256 id,
         uint256 oldState,
         uint256 newState,
-        uint256 isOldStateGenesis,
+        bool isOldStateGenesis,
```

Comment:
I'am generally ok with bool if publisher was verified to work properly with this input type.

Reply:
yes, updated publisher